src/libos/src/process/do_vfork.rs

```diff
@@ -87,6 +87,10 @@ fn restore_parent_process(mut context: *mut CpuContext, current_ref: &ThreadRef)
     let current_thread = current!();
     let current_pid = current_ref.process().pid();
 
+    // Todo: close all child created files, such as below.
+    // But parent opened files need to be filterd.
+    // current_thread.close_all_files();
```

Comment:
Please mention the "Drop for EpollFile" to help the one to fix it when vfork implementation updated.